src/helpers/hotkeysManager.js

```diff
@@ -460,7 +461,17 @@ WebViewer(...)
    * @ignore
    */
   createToolHotkeyHandler(handler) {
+    const { getState } = this.store;
+
     return (...args) => {
+      const openElements = selectors.getOpenElements(getState());
+      const currentToolName = core.getToolMode().name;
+    
+      if (currentToolName=== window.Tools.ToolNames.SIGNATURE && openElements['signatureModal']) {
+        // disable changing tool when the signature overlay is opened
+        return;
```

Comment:
Also I think we can just combine these two `if`s into this. Maybe something like
```javascript
if (isFocusingElement() || isSignatureModalOpen) {
  return;
}

handler(...args);
```

Reply:
I refactor this to only have one `if`